app/services/identity_linker.rb

```diff
@@ -25,7 +25,8 @@ def link_identity(
     scope: nil,
     verified_attributes: nil,
     last_consented_at: nil,
-    clear_deleted_at: nil
+    clear_deleted_at: nil,
+    email_address_id: nil
```

Comment:
Question: If a user has a single email when they connect to a partner, do we save the `email_address_id` corresponding to that email?

And if not, should we? Since a user may add new emails in the future, and they may not expect that the email they'd originally shared with the partner would change?

Reply:
It does also get saved for a single-email owner. 
I agree it should be.